### DISCUSSION TITLE
How to retrieve a previously saved password for a vault?

### DISCUSSION THREAD

**Discussion by dmitrydvornichenko** [sadv1r/ansible-vault-editor-idea-plugin#101] [Q&A] [answered]
By the way, I have more complicated question.
Is there a possibility to recover previously saved password for a vault that has been deleted or renamed?
Thank you for your attention.

**Comment by sadv1r** [comment, marked as answer]
It depends on the operating system. Here are the places where it can be stored: https://www.jetbrains.com/help/idea/reference-ide-settings-password-safe.html

Password name will begin with **IntelliJ Platform AnsibleVaultPass**
